app/dashboard/helpers.py

```diff
@@ -450,12 +450,17 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
                 latest_old_bounty_dict['bounty_reserved_for_user'] = Profile.objects.get(pk=latest_old_bounty_dict['bounty_reserved_for_user'])
             bounty_kwargs.update(latest_old_bounty_dict)
 
+
         try:
             new_bounty = Bounty.objects.create(**bounty_kwargs)
             new_bounty.fetch_issue_item()
             try:
                 issue_kwargs = get_url_dict(new_bounty.github_url)
                 new_bounty.github_issue_details = get_gh_issue_details(**issue_kwargs)
+
+                # If bounty is_featured flag is on
```

Comment:
I think this comment isn't needed since the code is so readable, what do you think?

Reply:
I'll remove the comment